meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8qm-mek/fw_env.config

```diff
@@ -1 +1 @@
-/mnt/boot/uboot.env     0x0000          0x2000
+/mnt/boot/uboot.env     0x0000          0x4000
```

Comment:
I prefer not to duplicate patches across PRs:
https://github.com/foundriesio/meta-lmp/pull/454/commits/f3843a561c692ba95751cc92fa2fac47da0dd778

Reply:
Sure, will drop it

Reply:
done